### ISSUE TITLE
Can you publish new version to npm?

### ISSUE THREAD

**Issue by ikalafat** [ldn0x7dc/react-native-transformable-image#12]
Hi @ldn0x7dc.

Can you publish new version (latest master with last commit) to npm? 

Also for react-native-transformable-view?

Thanks.

**Comment by andrekovac** [comment]
@ldn0x7dc Would be great if you update the npm version! I just implemented that change myself again because I also needed `onSingleTapConfirmed` & `onResponderSingleTapConfirmed` in your project [react-native-view-transformer](https://github.com/ldn0x7dc/react-native-view-transformer).

You really created some awesome projects here concerning your [react-native-gesture-responder
](https://github.com/ldn0x7dc/react-native-gesture-responder), this one and the aforementioned  [react-native-view-transformer](https://github.com/ldn0x7dc/react-native-view-transformer). Thanks a lot for your efforts!
However, you don't seem to be actively maintaining your projects anymore. Do you need some help maintaining them? Do you perhaps want to invite people to merge pull-requests? It looks like next to me, @ikalafat and @doochik are active users of your React Native gesture packages.
What do you guys think?

P.S. @ldn0x7dc 好喜欢你做的！加油~~

**Comment by ikalafat** [comment]
I agree, it would be cool if someone else also had write access to repos/npm. 

These libraries are awesome, they are being used, and it would be pity if they get forgotten.
